dev/fattest.simplicity/src/componenttest/topology/utils/MvnUtils.java

```diff
@@ -186,6 +190,26 @@ private static void populateJarsFromWlp(File pomXml) throws Exception {
         }
     }
 
+    public static int runTCKMvnCmd(LibertyServer server) throws Exception {
+        if (!init) {
+            init(server);
+        }
+        // Everything under autoFVT/results is collected from the child build machine
```

Comment:
not clear that logDir is 'autoFVT/results' dir

Reply:
Might be useful to have  a version of this method with the dirs/filenames as parameters and call it here- to enable multiple TCKs tests to be picked from in one FAT bucket that can call the lower level method.

Reply:
Also LOG_DIR has a specific different meaning as an environment variable: https://www.ibm.com/support/knowledgecenter/en/SSEQTP_8.5.5/com.ibm.websphere.wlp.doc/ae/rwlp_logging.html